vllm_spyre/envs.py

```diff
@@ -113,6 +113,12 @@ def _backend_backwards_compat() -> str:
     "VLLM_SPYRE_N_TOKENS_PREFILL_PRIO":
     lambda: int(os.getenv("VLLM_SPYRE_N_TOKENS_PREFILL_PRIO", "-1")),
 
+    # scheduling heuristic: maximal waiting (blocking) time for prefill
+    # Prefills waiting longer than VLLM_SPYRE_MAX_WAITING_TIME_PREFILL
+    # seconds will have priority after the current decode batch has finished.
+    "VLLM_SPYRE_MAX_WAITING_TIME_PREFILL":
```

Comment:
The name should reflect the units of time that are being used (e.g., `VLLM_SPYRE_MAX_WAITING_TIME_SECONDS`) or something. Should we also consider using an integer instead of a float?